Tests/ScipioKitTests/DescriptionPackageTests.swift

```diff
@@ -74,10 +74,12 @@ final class DescriptionPackageTests: XCTestCase {
         XCTAssertEqual(package.name, "TestingPackage")
 
         XCTAssertEqual(
-            Set(try package.resolveBuildProducts().map(\.target.name)),
+            try package.resolveBuildProducts().map(\.target.name),
             [
                 "Logging",
                 "TestingPackage",
+                "ExecutableTarget",
+                "MyPlugin"
             ]
```

Comment:
If you comment out the custom Hashable conformance above this test will fail when Scipio depends on the latest SwiftPM development snapshot.